### PR TITLE
blog: add New Stack post

### PR DESCRIPTION
**Pull request by TheTanc**

### CODE REVIEW

**Review by NicQuestDB — COMMENTED** [review]
Reflexions >> Thoughts

Tag's: is it important for SEO? If so, lets do something else than moore's law ?

File: blog/2020-09-24-peformance-matters.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+QuestDB is concerned with capturing time-series data in particular, which lets us represent and understand change over time. Time-series data might pertain to changes to the weather, changes in a machine’s performance, or even changes in your own weight. But quite unlike weighing yourself once a day and storing those standalone states in a database, time-series data calls for capturing every single tiny fluctuation in your weight, up or down, whenever you sweat, get sick, eat a meal, or use the bathroom.
+
+Processing this category of data calls for a high-performance system that can quickly manipulate lots of individual data points to turn that data into a decision-making aid. Performance is uniquely important to time-series data for the following reasons.
+
+# Time Series Data Is Explosive
```
Comment (mpsq) [inline blog/2020-09-24-peformance-matters.md:26]:
```suggestion
## Time series data is explosive
```

File: blog/2020-09-24-peformance-matters.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+It’s at the heart of connected devices, the Internet of Things, autonomous cars, financial services, and even server farm monitoring. Rather than capture a single data point, time-series data calls for capturing tens of thousands of data points. But it doesn’t even stop there — not only does time-series data keep growing and never stopping, but it can grow in bursts, generating lots of readings in a short amount of time.
+
+A weather station capturing time-series data about wind speed might record zero for a long time. But as soon as it gets windy, you’ll get thousands of measurements per second because the measurement is changing a lot. It takes a high-performing system to capture and record it effectively.
+
+Time-series data is everywhere a modern technologist looks, and the tools for managing it effectively are a little specialized. Otherwise, the exploding need for processing power and the simultaneously reduced availability of it becomes a supply and demand problem.
+
+# The End of Moore’s Law Is in Sight
```
Comment (mpsq) [inline blog/2020-09-24-peformance-matters.md:33]:
```suggestion
## The end of Moore’s law is in sight
```

File: blog/2020-09-24-peformance-matters.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+That’s why it’s time to focus on the other side of this equation: the software. The solution is to write leaner code that’s more hardware-efficient and extracts performance gains from having the software tuned so effectively. This kind of software is less reliant on the hardware side for its capabilities, opens up new possibilities for harnessing data, and operates in realtime without any lag.
+
+Moore’s law is approaching its physical limit because it’s only possible to fit so many transistors into an integrated circuit — most of the performance to data has come from hardware, indirectly giving developers permission to write lazy, bloated code. But there is much less room for optimizing hardware these days. Chip manufacturers are approaching a time when they’d need a new physics to improve on today’s modern processors. That means it’s time to focus on improving the software.
+
+# It Reduces Your Cloud Bills
```
Comment (mpsq) [inline blog/2020-09-24-peformance-matters.md:40]:
```suggestion
## It reduces your cloud bills
```

**Review by mpsq — CHANGES_REQUESTED** [review]
minor tweaks

File: blog/2020-09-24-peformance-matters.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: Why performance matters in time-series data
+author: Nicolas Hourcard
+author_title: QuestDB Team
+author_url: https://github.com/NicQuestDB
+author_image_url: https://avatars.githubusercontent.com/NicQuestDB
+description:
+  Reflexions on why speed and performance are crucial to time-series databases ingestion and analysis.
```
Comment (mpsq) [inline blog/2020-09-24-peformance-matters.md:8]:
```suggestion
  Thoughts on why speed and performance are crucial to time-series databases ingestion and analysis.
```

File: blog/2020-09-24-peformance-matters.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+tags: [big data, moore's law]
+---
+
+Reflexions on why speed and performance are crucial to time-series databases ingestion and analysis, originally posted in [The New Stack](https://thenewstack.io/why-performance-matters-in-time-series-data/).
```
Comment (mpsq) [inline blog/2020-09-24-peformance-matters.md:12]:
```suggestion
Thoughts on why speed and performance are crucial to time-series databases ingestion and analysis, originally posted in [The New Stack](https://thenewstack.io/why-performance-matters-in-time-series-data/).
```

**Review by mpsq — CHANGES_REQUESTED** [review]

**Review by mpsq — APPROVED** [review]

**Pull request merged**